functional_tests/scylla_operator/test_functional.py

```diff
@@ -825,3 +825,21 @@ def test_cloud_bundle_connectivity_cassandra_stress(tester):
 
     assert "latency 99th percentile" in output[0]
     assert float(output[0]["latency 99th percentile"]) > 0
+
+
+@pytest.mark.required_operator("v1.8.0")
+@pytest.mark.requires_tls
+def test_cloud_bundle_connectivity_scylla_bench(tester):
+
+    assert tester.db_cluster.connection_bundle_file, "cloud bundle wasn't found"
+
+    cmd = (
+        "scylla-bench -workload=sequential -mode=write -replication-factor=1 -partition-count=10 "
+        "-clustering-row-count=5555 -clustering-row-size=uniform:10..20 -concurrency=10 "
+        "-connection-count=10 -consistency-level=one -rows-per-request=10 -timeout=60s -duration=1m"
+    )
+
```

Comment:
Maybe we want to check that the bundle was really used?
For example, we can search for the bundle presence in the output like we do for the "Clustering row size".

Reply:
yes, we could add some checks

Reply:
I don't see it in the latest change.
Do you plan to add it?

Reply:
We don't have any indication in the output, it's something we'll need to add to scylla-bench.

Reply:
so no, not right now.

maybe when we'll need to switch the gocql driver in scylla-bench (since we need that change to connect), we'll need the connection details to the log

Reply:
Ok, then "TODO" comment would be useful in this case.